platformio_core_defs.ini

```diff
@@ -245,10 +245,10 @@ build_flags                 = -DESP32_STAGE
                               -include "ESPEasy_config.h"
 
 [core_esp32_stage]
-platform                    = https://github.com/Jason2866/platform-espressif32.git
-platform_packages           = framework-arduinoespressif32 @ https://github.com/tasmota/arduino-esp32/releases/download/2.0.1/framework-arduinoespressif32-release_IDF4.4.tar.gz
-                              platformio/tool-esptoolpy @ https://github.com/tasmota/esptool/releases/download/v3.2/esptool-v3.2.zip
+platform                    = https://github.com/tasmota/platform-espressif32/releases/download/v2.0.4.1/platform-espressif32-2.0.4.1.zip
```

Comment:
Why 2.0.4.1 and not 2.0.5.1?

Reply:
Hmm, must have mixed up a merge conflict 😞

Reply:
But AFAICS, that definition isn't currently used 😅

Reply:
You sure? ;)
https://github.com/letscontrolit/ESPEasy/blob/86cd66816f538b4505f6d3f38734c18d3db52251/platformio_core_defs.ini#L236

Reply:
The merge error is in the `[core_esp32_stage]` section, in the `[core_esp32_IDF4_4__2_0_5]` section that is referenced from current `env:` sections the correct version is used.